app/routes/capital-projects.$managingCode.$capitalProjectId.tsx

```diff
@@ -1,3 +1,41 @@
+import { LoaderFunctionArgs, json } from "@remix-run/node";
+import { useLoaderData, useNavigate } from "@remix-run/react";
+import {
+  findCapitalProjectByManagingCodeCapitalProjectId,
+  findAgencies,
+} from "../gen";
+import { CapitalProjectDetailPanel } from "../components/CapitalProjectDetailPanel";
+
+export async function loader({ params }: LoaderFunctionArgs) {
+  const agenciesResponse = await findAgencies({
+    baseURL: `${import.meta.env.VITE_ZONING_API_URL}/api`,
+  });
+  if (
+    typeof params.managingCode === "undefined" ||
+    typeof params.capitalProjectId === "undefined"
+  ) {
+    throw json("Bad Request", { status: 400 });
+  }
+  const capitalProject = await findCapitalProjectByManagingCodeCapitalProjectId(
+    params.managingCode,
+    params.capitalProjectId,
+    {
+      baseURL: `${import.meta.env.VITE_ZONING_API_URL}/api`,
+    },
+  );
+  return json({ capitalProject, agencies: agenciesResponse.agencies });
+}
+
 export default function CapitalProject() {
-  return <></>;
+  const navigate = useNavigate();
+  const { capitalProject, agencies } = useLoaderData<typeof loader>();
+  return (
+    <CapitalProjectDetailPanel
+      capitalProject={capitalProject}
+      agencies={agencies}
+      onClose={() => {
+        navigate("/");
```

Comment:
This will clear any query parameters and reset the state of the geography menu. My understanding is that we're trying to decouple the content panel from the geography menu. To honor this, we should preserve the query parameters. The capital projects layer has an example of how to [pass the existing parameters to the new route](https://github.com/NYCPlanning/ae-cp-map/blob/62a9766562b40b5a31c99067f6ae1f082479b4f4/app/components/layers/useCapitalProjectsLayer.client.tsx#L48).